DfciPkg/Docs/Dfci_Feature.md

```diff
@@ -9,8 +9,8 @@ that are typically configured within a BIOS menu by a human.
 High value configuration can be moved to UEFI BIOS where it is resilient against malware,
 rootkits, and non-persistent physical tampering.
 Whereas traditional UEFI security implementations required a physical touch, DFCI securely
-enables **zero-touch** remote configuration of these settings built upon[Microsoft Intune](
-https://docs.microsoft.com/en-us/intune/configuration/device-firmware-configuration-interface-windows)
+enables **zero-touch** remote configuration of these settings built upon[Microsoft Intune]

```

Comment:
The link format before should have been valid. The new ones don't render properly.

This particular link doesn't have a space between "upon" and the link so that might cause the linter to be confused.

Did all of these get flagged?

Reply:
Fresh tools in local CI resulted in the following:


INFO - DfciPkg/Docs/Dfci_Feature.md:13:1 MD034/no-bare-urls Bare URL used [Context: "https://docs.microsoft.com/en-..."]
INFO - DfciPkg/Docs/Dfci_Feature.md:31:1 MD034/no-bare-urls Bare URL used [Context: "https://www.microsoft.com/en-u..."]
INFO - DfciPkg/Docs/Dfci_Feature.md:66:1 MD034/no-bare-urls Bare URL used [Context: "https://docs.microsoft.com/en-..."]
INFO - DfciPkg/Docs/Internals/DfciInternals.md:306:1 MD034/no-bare-urls Bare URL used [Context: "https://github.com/microsoft/m..."]

The updates allowed local CI to pass.

Reply:
What version of markdownlint are you using? I didn't see an error running against the main branch tip (e1228ea) using v0.31.1 (version used in CI).

Reply:
0.33.0 was installed. I'll change version, back to 0.31.1, and rerun (and revert the markdown changes)